src/dsp_tools/utils/excel_to_json/utils.py

```diff
@@ -0,0 +1,289 @@
+# pylint: disable=wrong-import-order
```

Comment:
It is a good practice to sort the imports:

1. python stdlib
2. third-party-packages from pip
3. dsp-tools internal packages

In VSCode, the `isort` extension offers auto-fix support for that.

Reply:
But thinking again about this topic, I wonder if we should care about import sorting at all. If you want, we can just ignore the "wrong-import-order" globally (in `pyproject.toml`), and then never care again about import sorting. In the past, I got used to thinking that sorting imports would be good practice, but I wonder if it is worth spending time with it.

Reply:
Done